pom.xml

```diff
@@ -113,7 +113,7 @@
         <!-- MicroProfile specs  -->
         <config-version>2.0</config-version>
         <ft-version>3.0</ft-version>
-        <health-version>3.0</health-version>
+        <health-version>3.1-RC3</health-version>
```

Comment:
also need a copyright year update at the top of the file

Reply:
Good catch, thanks